README.md

```diff
@@ -1,16 +1,32 @@
-# Slack Orb in Go [![CircleCI Build Status](https://circleci.com/gh/CircleCI-Public/slack-orb.svg?style=shield "CircleCI Build Status")](https://circleci.com/gh/CircleCI-Public/slack-orb) [![CircleCI Orb Version](https://badges.circleci.com/orbs/circleci/slack.svg)](https://circleci.com/orbs/registry/orb/circleci/slack) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/circleci-public/slack-orb/master/LICENSE) [![CircleCI Community](https://img.shields.io/badge/community-CircleCI%20Discuss-343434.svg)](https://discuss.circleci.com/c/ecosystem/orbs)
+# Slack Orb for CircleCI [![CircleCI Build Status](https://circleci.com/gh/CircleCI-Public/slack-orb.svg?style=shield "CircleCI Build Status")](https://circleci.com/gh/CircleCI-Public/slack-orb) [![CircleCI Orb Version](https://badges.circleci.com/orbs/circleci/slack.svg)](https://circleci.com/orbs/registry/orb/circleci/slack) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/circleci-public/slack-orb/master/LICENSE) [![CircleCI Community](https://img.shields.io/badge/community-CircleCI%20Discuss-343434.svg)](https://discuss.circleci.com/c/ecosystem/orbs)
 
-This repository houses a conceptual prototype demonstrating the potential appearance and functionality of the CircleCI [Slack orb](https://github.com/CircleCI-Public/slack-orb), reimplemented in Go. Please note that this version is exploratory and not intended to supplant the original orb, nor is it advisable to utilize this in production environments. Explore, understand, and experiment, but proceed with caution when integrating it into practical applications. 🛠✨
+The Slack Orb for CircleCI connects your CI/CD pipelines to Slack for ChatOps. Fully customizable, platform agnostic, and easy to integrate into your new or existing CI/CD pipelines.
+
+## Slack Orb v5+ Breaking Changes
+
+The Slack Orb for CircleCI is now written in Go! This means the orb is now more easily maintained, and better supports a wide range of platforms, no longer requiring BASH, CURL, JQ, or any other shell tools or commands.
+
+1. The orb now downloads the Slack Orb CLI binary to `.circleci/orbs/circleci/slack/$PLATFORM/$ARCH` in the working directory.
+   1. This can be checksum verified
+   2. This is cacheable if desired.
+2. You can no longer use evaluated sub-shell commands (e.g. `$(date +%s)`).
+   1. Instead, you can pre-populate environment variables in the `$BASH_ENV` file. [See more in our wiki]()
+   2. The Slack Orb has a small number of "built in" environment variables for use in templates [See more in our wiki]()
+3. On-hold job removed. Low usage prompted removal from this release for simplicity of the documentation. We will revisit this as feedback comes in.
```

Comment:
A few more breaking changes that should be mentioned:

- The `custom` parameter was replaced with `template_inline`.
- The `template` parameter is reserved for built-in templates now and it won't support dynamic templates anymore.
- Dynamic templates must be imported via the `template_var` parameter.
- Sub-shell commands inside $BASH_ENV will no longer be expanded when using Alpine-based images.

Reply:
Thank you!